logic/scripts/update_token_insight.py

```diff
@@ -427,13 +427,13 @@ def alert_on_balance_drop(wallet, label, eth_threshold):
 def fetch_reserved_wallet_balances():
     print("Computing OGN stats...")
     # Fetch reserved wallet balances
-    # fetch_wallet_balance(token_stats.foundation_reserve_address)
+    fetch_wallet_balance(token_stats.foundation_reserve_address)
```

Comment:
Why was this line commented and the ones below out in the first place? Was it during development and we forgot to uncomment?

Reply:
I think it might have been commented out in error, because the balance is then used later on to calculate circulating supply. It doesn't matter much with the old foundation reserve address because I don't think it's in use anymore, but the old team and investor addresses still have a balance

Reply:
Thanks for checking.